src/pages/HomeView/index.js

```diff
@@ -23,6 +25,8 @@ export default function HomeView(props) {
       <Hero />
       <div id="back-to-top-anchor"></div>
       <NonHero />
+      <Search />
+      <Recommended />
```

Comment:
Instead of 7 cards here:
https://github.com/codeforcauseorg/edu-client/blob/a6bbc047e022ec132d14b85e01c86fe4cd07eb9c/src/pages/HomeView/NonHero.js#L24

Show only 1 or 2

Reply:
Take Search and Recommended inside `<NonHero/>`.

Reply:
If you are getting extra marginleft added due to container, just add some -ve marginleft

Reply:
I think we should add a horizontal scroll to these cards too, What you say?

Reply:
Yeah, for sure. You can create a new PR and make your changes